cmd/promtool/main.go

```diff
@@ -283,9 +297,38 @@ func main() {
 	}
 }
 
+var lintError = fmt.Errorf("lint error")
```

Comment:
IMO this naming makes more sense than what revive wants, so we can ignore it.

```suggestion
// nolint:revive
var lintError = fmt.Errorf("lint error")
```